device-types/Eaton/9SXEBM180RT.yaml

```diff
@@ -0,0 +1,23 @@
+---
+manufacturer: Eaton
+model: 9SXEBM180RT
+slug: eaton-9sxebm180rt
+part_number: 9SXEBM180RT
+u_height: 3
+is_full_depth: true
+airflow: front-to-rear
+comments: Eaton 9PX extended battery moduel (EBM), 180V, Rack/tower, 3U [Manual](https://www.eaton.com/content/dam/eaton/products/backup-power-ups-surge-it-power-distribution/backup-power-ups/eaton-9px-ups/user-guides/eaton-9sx-9px-ups-5-6-kva-ebm-180v-installation-and-user-manual.pdf)
+power-ports:
+  - name: UPS Input
+    type: other
+interfaces:
+  - name: Battery connector, top
+    type: other
+  - name: Battery detection, left
+    type: other
+    label: Batt detection
+  - name: Battery connector, down
+    type: other
+  - name: Battery detection, right
+    type: other
+    label: Batt detection
```

Comment:
Once again these are not meant for this, they are meant for networking interfaces. Maybe define these as `module-bays` and create the associated batteries as modules